doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='bus-app-con-rest-proc_{context}']
+= Configuring REST endpoints for Spring Boot applications
+
+After you create your Spring Boot project, you can configure  the host, port, and path for the REST endpoint for your Spring Boot application.
+
+.Prerequisites
+* You have a ZIP file that you created using the http://start.jbpm.org[business application]  website.
+
+.Procedure
+. Unzip the `<business-application>.zip` file and navigate to the `<business-application>/<business-application>-service/src/main/resources` folder.
```

Comment:
Extract. Never Unzip. Trademark issue.

Reply:
If you look in the path, that content is outdated. It was updated in another then carried to this one when I rebased.